index.html

```diff
@@ -89,10 +93,10 @@ <h2 class="white-1 roboto-f f-weight700 l-height-24 f-size-26">I am a software d
 
     <div class="desktopversion">
```

Comment:
- [ ] Your Desktop version appears to have the hamburger button. Kindly replace that with the desktop menu to match the original design.

![image](https://user-images.githubusercontent.com/98453979/194642208-51e7a2a5-950a-4439-bb99-321b624352be.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/194642525-359be19a-6210-4b37-9671-2abd9e4c05e6.png)

- [ ] On the project section in the original design, you will notice that only the first card has the `see project`  button but I noticed all your cards have it. Kindly fix that to match the design.

![image](https://user-images.githubusercontent.com/98453979/194643468-c1586314-fd8b-44fe-849d-e47d6877edf8.png)

- [ ] On the contact section, Kindly adjust the space between the inputs to match the Figma design. Currently, its too big

![image](https://user-images.githubusercontent.com/98453979/194644456-19c1142d-f527-4bb9-a0b4-786e8a40180b.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/194644545-585396c7-d92f-4cb2-a4f6-8efb05fa62d3.png)